README.md

```diff
@@ -1,20 +1,34 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+A jQuery method that registers an event when a button or link is clicked. You can then do different things when a button is clicked
+
+(b) Where in your JS file should you put it, and WHY?
+It should go in your namespace and then called within $(document).ready
+
+I put it inside of a function - inside of a namespace for the following reasons:
+1. The namespace protects your variables
+2. The namespace init method gets called in document.ready
+3. You want to wait until the page is finished loading (and all your JS stuff is done) before being able to click on a button
 
-(b) Where in your JS file should you put it, and WHY? 
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+
+It means you are not pointing to jquery (the jquery script hasn't been sourced before your js files possibly - or not referencing it altogether)
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+
+App.init (well, the initialization function/method of your namespace)
```

Comment:
click handlers should also go in `$(document).ready()` - you wrote that correctly in Q1